src/test/unit_tests/test_bucketspace_versioning.js

```diff
@@ -639,7 +640,63 @@ mocha.describe('bucketspace namespace_fs - versioning', function() {
 
             const get_res = await s3_uid6.getObject({ Bucket: content_dir_bucket_name, Key: key });
             assert.equal(get_res.VersionId, res2.VersionId);
+        });
+
+        mocha.it('content directory - object tagging', async function() {
+            const dir_tagging_key = 'dir_tagging/';
+            const tag_set1 = { TagSet: [{ Key: "key1", Value: "Value1" }] };
+            const tag_set2 = { TagSet: [{ Key: "key2", Value: "Value2" }] };
 
+            await s3_uid6.putBucketVersioning({ Bucket: content_dir_bucket_name, VersioningConfiguration: { MFADelete: 'Disabled', Status: 'Enabled' } });
```

Comment:
after validating the behavior on AWS, can you add a test for suspended mode as well?